cmd/operator-sdk/up/local.go

```diff
@@ -112,7 +114,18 @@ func upLocal() error {
 		extraArgs := strings.Split(operatorFlags, " ")
 		args = append(args, extraArgs...)
 	}
-	dc := exec.Command(outputBinName, args...)
+
+	var dc *exec.Cmd
+
+	if debugFlag {
+		debugArgs := []string{"--listen=:2345", "--headless=true", "--api-version=2", "exec", outputBinName, "--"}
```

Comment:
@mathianasj I think the CI failure is a flake.
But you still need to pass `args` to the `debugArgs` I believe.